src/app/about-modal/about-modal.component.less

```diff
@@ -1,6 +1,6 @@
-@import (reference) '../../assets/stylesheets/_base';
-.modal-dialog-f8 { width: 696px; }
-.about-modal-f8 {
+@import (reference) '../../assets/stylesheets/shared/osio.less';
+.io-modal-dialog { width: 696px; }
```

Comment:
Why did you change this? The namespace we are using is `f8`. `io` is too generic.